frontend/src/utils/UIEditor/diffTable.ts

```diff
@@ -1,33 +1,82 @@
-import { detailedDiff } from "deep-object-diff";
-import { cloneDeep } from "lodash-es";
-import { CreateTableContext, Table } from "@/types";
+import { isEqual, isUndefined } from "lodash-es";
 import {
-  transformColumnToAddColumnContext,
-  transformTableToCreateTableContext,
-} from "./transform";
+  AlterTableContext,
+  CreateTableContext,
+  DropTableContext,
+  Table,
+} from "@/types";
+import { UNKNOWN_ID } from "@/types/const";
+import { transformTableToCreateTableContext } from "./transform";
+import { diffColumnList } from "./diffColumn";
 
 export const diffTableList = (
```

Comment:
I've added some comments for the diff functions in frontend. 

As for why we implement our own diff, since we don't use any 3rd party packages, I'm commenting on it in this PR's description.